modules/test/playwright/pages/site-admin-web/SiteSettingsLocalizationPage.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+import {SiteSettingsPage} from './SiteSettingsPage';
+
+export class SiteSettingsLocalizationPage {
+	readonly customDefaultLanguageOption: Locator;
+	readonly defaultLanguageOption: Locator;
+	readonly defaultLanguageSingleSelect: Locator;
+	readonly page: Page;
+	readonly saveButton: Locator;
+	readonly siteSettingsPage: SiteSettingsPage;
+
+	constructor(page: Page) {
+		this.customDefaultLanguageOption = page.getByLabel(
+			'Define a custom default language and additional available languages for this site.'
+		);
+		this.defaultLanguageOption = page.getByLabel(
+			'Use the default language options.'
+		);
+		this.defaultLanguageSingleSelect = page.locator(
+			`select[name="_com_liferay_site_admin_web_portlet_SiteSettingsPortlet_TypeSettingsProperties--languageId--"]`
```

Comment:
Hey @rebsilva, regarding you comment [here](https://github.com/liferay-objects/liferay-portal/pull/3353#discussion_r1606684724), I was talking about line 27. It passes a CSS seletor to  `page.locator()`, but we should always try to use Locators, with capital case, as seen [here](https://playwright.dev/docs/locators). Whenever people use `page.locator()`, I ask if they had already tried using Locators. If they did, that's fine, go on with `page.locator()`. If not, please try it, as they are more resilient to changes in the DOM structure.

Reply:
i see! i tried to use the Locators but i couldn't get it; i will double check again, but i'm almost certain that in this case is not possible, only if we change the code of the application

Reply:
This is the kind of situation where changing the code of the application can be a good thing. When Locators are not possible, it can be because of bad practices that may impact accessibility in a bad way, but we should always consider effort and benefits to do so. In this case, it's another team's module, so we would be blocking our tests to wait for another team to change the application, or to approve out change to their application, I don't think this makes sense here. As I said before, if you already tried using a Locator and it did not work, leave it like this. It's fine.

Reply:
i totally agree with you!